src/app/services/videoprocessingcuda/src/stream/input/CameraConfig.cpp

```diff
@@ -0,0 +1,17 @@
+#include "CameraConfig.h"
```

Comment:
Maybe we could inline both constructor, since there no code in the implementation file.

Reply:
Good point